common/polar_grid/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ endif()
 find_package(ament_cmake_auto REQUIRED)
 ament_auto_find_build_dependencies()
 
-find_package(Qt5 ${rviz_QT_VERSION} EXACT REQUIRED Core Widgets)
+find_package(Qt5 ${rviz_QT_VERSION} REQUIRED Core Widgets)
```

Comment:
It seems we should set `rviz_QT_VERSION` correctly.

Reply:
Certainly. I drop this PR